peps/pep-0768.rst

```diff
@@ -294,6 +294,26 @@ An example usage of the API would look like:
     except Exception as e:
         print(f"Failed to execute code: {e}")
 
+Configuration API
+-----------------
+
+To allow redistributors, system administrators, or users to disable this
+mechanism, several methods will be provided to control the behavior of the
+interpreter:
+
+A new environment ``PYTHON_DISABLE_REMOTE_DEBUGGING`` environment variable will
```

Comment:
Depending on how you feel, I _slightly_ prefer `DISABLE_REMOTE_DEBUG` (and likewise for the other uses) over debugGING. Saves a few characters, and I always find ING-words to be weird feature names.

Probably just my own bias though. Might be an American intonation thing as well ("remote DEE-bug") - I mostly speak to Americans about debugging 😆